Mathlib/FieldTheory/PrimitiveElement.lean

```diff
@@ -52,7 +56,7 @@ variable (F : Type*) [Field F] (E : Type*) [Field E] [Algebra F E]
 
 /-- **Primitive element theorem** assuming E is finite. -/
 theorem exists_primitive_element_of_finite_top [Finite E] : ∃ α : E, F⟮α⟯ = ⊤ := by
```

Comment:
Many theorems in this file could have `⊤` replaced with an arbitrary intermediate field, which would make them more easily applicable, even though there's no new mathematical content. Maybe in a future PR?

Reply:
I'm not sure since I haven't read that code before; it was already here when I add new contents.

Reply:
Yes, I think `primitive_element_inf_aux_of_finite_intermediateField` is a special case of `exists_primitive_element_of_finite_intermediateField` if the latter is changed a slightly. For others I think it's better for a future PR/issue.